### DISCUSSION TITLE
Prompt for multiple entries (Form).

### DISCUSSION THREAD

**Discussion by vaskoZGU** [Budibase/budibase#11362] [Help]
Hello,
Could you please help me with the following:

1. User fill a form and click Save
2. On click, a popup window (prompt) asks if the user wants to continue (with a new form) or not (Yes/No)
3. If No is clicked, save the form and close the window
4. If Yes is clicked, save the form and open it again, preserving part of the fields from the previous entry

Thank you in advance.

**Comment by melohagan** [comment]
Hey @vaskoZGU 

This is not quite the use-case you described, but I believe it is fairly close: https://docs.budibase.com/docs/save-two-linked-tables-in-one-form

**Reply by vaskoZGU** [reply]
Thank you @melohagan,

I'll check.
